control/controlclient/direct.go

```diff
@@ -352,6 +355,7 @@ func (c *Direct) doLogin(ctx context.Context, t *oauth2.Token, flags LoginFlags,
 	if url != "" {
 	} else if regen || persist.PrivateNodeKey.IsZero() {
 		c.logf("Generating a new nodekey.")
+		systemd.Status("generating new nodekey")
```

Comment:
I think we can omit this line entirely. This should only be in this state for a fraction of a millisecond.